webhooks.md

```diff
@@ -304,16 +304,38 @@ If you would like to run the operator locally, extra steps must be taken to eith
 
 Before following either section below, make sure you're pointing to a valid `KUBECONFIG` or have otherwise used `oc` to log into your cluster.
 
+**NOTE for FR2 and later:** These options only work if the operator you wish to run locally has been installed individually via OLM.  If it was installed via the
+OpenStack operator, this does not work (unless, of course, you're trying to run the OpenStack operator itself locally, which is fine).
+
 ### Disabling webhooks
 
 You can disable the webhooks if you don't need them for your local dev/testing purposes.  The procedure is as follows:
 
+**FR1 and earlier**
+
 1. If you installed the operator via OLM, remove its webhook definitions from its CSV:
 
 ```bash
 oc patch csv -n openstack-operators <your operator CSV> --type=json -p="[{'op': 'replace', 'path': '/spec/webhookdefinitions', 'value': []}]"
 ```
 
+**FR2 and later**
+
+1. If this is a service operator (i.e. is not OpenStack operator) installed via OLM, remove its webhook definitions from its CSV:
+
+```bash
+oc patch csv -n openstack-operators <your operator CSV> --type=json -p="[{'op': 'replace', 'path': '/spec/webhookdefinitions', 'value': []}]"
+```
```

Comment:
These commands are the same, but the reasoning for them is different between the RHOSO versions.